changelog/issue-4226.md

```diff
@@ -0,0 +1,5 @@
+audience: developers
+level: patch
+reference: issue 4226
+---
+The `yarn generate` command no longer performs minifcation of `yarn.lock` files, so that automatic dependency upgrade PRs will succeed.  Run `yarn minify` to do this manually.
```

Comment:
```suggestion
The `yarn generate` command no longer performs minification of `yarn.lock` files, so that automatic dependency upgrade PRs will succeed.  Run `yarn minify` to do this manually.
```

or maybe reworded

```suggestion
The `yarn generate` command no longer combines redundant lines in `yarn.lock` files, so that automatic dependency upgrade PRs will succeed.  Run `yarn minify` to do this manually.
```